### DISCUSSION TITLE
Set cell value with condition

### DISCUSSION THREAD

**Discussion by TTutorials** [PHPOffice/PhpSpreadsheet#2460] [Q&A] [answered]
How to modify the cell value depend on id from another cell?
Something like if A1 = foo, and set cell value on B1 = bar

**Comment by MarkBaker** [comment, marked as answer]
Use a formula perhaps?
```
$worksheet->setCellValue('B1', '=IF(A1="bar", "foo", "not foo")');
```

Otherwise it's going to be a case of implementing the logic in your own code
```
if ($worksheet->getCell('A1)->getValue() === 'foo') {
    $worksheet->setCellValue('B1', 'bar');
}
```
